app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+from shutil import move
+from os import makedirs
+from os.path import dirname
+
+
+def move_file(command: str) -> None:
+    params = command.split()
```

Comment:
try to unpack this like this `arg1,arg2,arg3 = command.split() `

Reply:
Again, I'll have to handle possible exception here if I do that

Reply:
agree, but u can unpack this after ur condition 
arg1, arg2, arg3 = params
it seems to me, that for better readability is is better to use vars instead of indexes

Reply:
but it is up to u

Reply:
Yeah, that was the solution I was proposing in the previous ("copy file") project you've rejected as my compromise.